client/src/components/App.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import axios from 'axios';
+
+import styles from './styles.css';
+
+import sampleData from '../../../database/sampleData.js';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      date: '',
+      time: '7:00 pm',
+      partySize: 2
+    }
+
+    this.partySizes = ['1 person'];
```

Comment:
is the array defaulted to show 1 person? and then when genratePartSizes gets called , it builds up the array of party size choices with 'one person' on the front.. just understanding your code thats all.

Reply:
That's exactly how it works! The database tracks one bit of data related to party size, and that's an integer that represents the maximum party size. So on the client side, when it gets that number, it just builds an array of all the party size options up to that max, to be used in the selector.

Reply:
ahh nice. makes sense. very elegant